angular/src/app/components/modal-questionnaire-viewer/modal-questionnaire-viewer.component.styl

```diff
@@ -19,1403 +19,174 @@
-    pointer-events: all;
-    cursor: pointer;
     position: relative;
+    width: 100%;
+    height: 500px;
 }
 
-::ng-deep div.addDeclineOptionWrapper {
-    cursor: default;
+::ng-deep p.locationQuestionDetails {
+    font-weight: bold;
 }
 
-
-/*DROP DOWN MENU VIEW FOR MULTIPLE CHOICE QUESTIONS*/
-
-::ng-deep div.dropDownViewModeContainer {
-    position: relative;
+::ng-deep textarea.locationAddress {
     display: block;
-    text-align: left;
-}
-
-::ng-deep div.dropDownViewQuestionContainer {
-    padding: 10px !important;
     position: relative;
-    display: inline-block;
-    width: auto;
-    min-width: 400px;
-    border: 1px solid hsl(0, 0%, 80%);
-    border-radius: 5px;
-    left: 0;
-}
-
-::ng-deep div.buttonQuestionnaireDropDown {
-    display: inline-block;
-    position: absolute;
-    top: 20px;
-    right: 20px;
-    cursor: default;
-    pointer-events: none;
-    width: 0;
-    height: 0;
-    border-left: 7px solid transparent;
-    border-right: 7px solid transparent;
-    border-top: 10px solid hsl(0, 0%, 0%);
-}
-
-::ng-deep .question-view-mode div.div.buttonQuestionnaireDropDown {
-    display: none;
+    width: 100%;
+    height: 80px;
 }
 
-
-
-::ng-deep div.questionnairePlaceholder {
-    padding: 0 10px;
-    border-radius: 5px;
-    border: 1px solid hsl(0, 0%, 80%);
-    position: relative;
-    min-height: 50px;
-    min-width: 400px;
-    display: block;
-    font-size: 14pt;
-    color: hsl(0, 0%, 20%);
-    vertical-align: middle;
+::ng-deep app-modal-questionnaire-viewer > div.view-only-questionnaire-container > img {
     margin-bottom: 10px;
 }
 
-::ng-deep div.dropDownViewModeQuestion {
-    display: inline-block !important;
-    width: auto;
-    padding: 5px !important;
-    border-radius: 5px !important;
-    border:  1px solid hsl(0, 0%, 0%) !important;
-}
-
-::ng-deep div.answerBubbleViewMode, ::ng-deep div.answerBubbleViewModeSelected, ::ng-deep div.questionOptionShape {
-    position: relative;
-    line-height: 50px;
-    display: inline-block;
-    width: 24px;
-    height: 24px;
-    /*margin-top: 13px;*/
-    border: solid 1px hsl(0, 0%, 10%);
-}
-
-::ng-deep .question-view-mode div.answerBubbleViewMode {
-    margin: 0 !important;
-}
-
-::ng-deep div.questionOptionShape {
-    position: absolute;
-    display: inline-block;
-    border-radius: 15px;
-    height: 28px;
-    width: 28px;
-    margin-right: 3px;
-    top: 50%;
-    margin-top: -14px
-}
-
-::ng-deep div.answerBubbleViewModeSelected {
-    background-color: hsl(200, 100%, 55%);
-}
-
-
-/*MODIFIERS*/
-
-::ng-deep textarea.longAnswer {
-    height: 160px !important;
-}
-
-::ng-deep textarea.mediumAnswer {
-    height: 100px !important;
-}
-
-::ng-deep textarea.shortAnswer {
-    height: 52px !important;
-}
-
-
-/* QUESTION SELECTION CLASSES*/
-
-::ng-deep .optionSelected {
-    background-color: rgb(51, 204, 255) !important;
-}
-
-
-::ng-deep div.hidden {
-    display: none;
+::ng-deep div.locationMapContainer,
+::ng-deep img {
```

Comment:
all `img`'s are getting the border (i.e in leaflet or on main page).

it looks like the following would fix it:

```
::ng-deep div.locationMapContainer,
::ng-deep app-modal-questionnaire-viewer > div.view-only-questionnaire-container > img {
``` 

but it also appears that dropping the `ng-deep` would fix it as well.  not certain on that, as i understood that we needed the `ng-deep` everywhere:
```
::ng-deep div.locationMapContainer,
img {
```

Reply:
> but it also appears that dropping the ng-deep would fix it as well. not certain on that, as i understood that we needed the ng-deep everywhere:

ignore the dropping the ng-deep suggestion.  seems like specificity is different than out angular components so ng-deep is probably needed in some places so probably best to leave alone.

Reply:
Yeah, the ng-deep is needed to override those deeply nested elements in the questionnaire... its a bit wonky. Can't wait to convert this thing to a legit proper component with isolated styling control.

Reply:
I'll add the specificity to avoid adding borders to images in the other pages. Didn't mean to touch any design above the questionnaire level, but I did forget to test the unrelated parts of the app.

Reply:
Okay, I better isolated the selectors to only effect the questionnaire viewer modal. Should be much better now.